tianshou/highlevel/experiment.py

```diff
@@ -442,6 +484,46 @@ def build(self) -> Experiment:
         )
         return experiment
 
+    def build_default_seeded_experiments(self, num_experiments: int) -> dict[str, Experiment]:
+        """Creates a dict of experiments with non-overlapping seeds, starting from the configured seed.
+
+        The keys of the dict are the experiment names, which are derived from the seeds used in the experiments.
+        """
+        configured_experiment_config = copy(self.experiment_config)
+        configured_experiment_seed = configured_experiment_config.seed
+        configured_sampling_config = copy(self.sampling_config)
+        configured_train_seed = configured_sampling_config.train_seed
+        num_train_envs = configured_sampling_config.num_train_envs
+
+        seeded_experiments = {}
+        for i in range(num_experiments):
+            experiment_seed = configured_experiment_seed + i
+            new_experiment_config_dict = asdict(configured_experiment_config)
+            new_experiment_config_dict["seed"] = experiment_seed
+
+            new_train_seed = i * num_train_envs + configured_train_seed
+            new_sampling_config_dict = asdict(configured_sampling_config)
+            new_sampling_config_dict["train_seed"] = new_train_seed
+
+            self.experiment_config = ExperimentConfig(**new_experiment_config_dict)
+            self.sampling_config = SamplingConfig(**new_sampling_config_dict)
+            exp = self.build()
+
+            full_name = ",".join(
+                [
+                    f"experiment_seed={exp.config.seed}",
+                    f"train_seed={exp.sampling_config.train_seed}",
+                    f"test_seed={exp.sampling_config.test_seed}",
+                ],
+            )
+            experiment_name = shortener(full_name, 4)
```

Comment:
The experiment's path is composed of two parts. The persistence_dir in the experiment_config and a name given to run() of the Experiment. The name created here is just a convenience suggestion to have a (somewhat) meaningful default. It can obviosuly just be a list, too, and the user has to come up with their own names